packages/test/src/util/server-tests-runner/index.js

```diff
@@ -60,7 +60,7 @@ exports.run = function(allTests, options) {
     const convertedMochaArgs = convertMochaConfigToArgs(mochaOptions);
 
     if (convertedMochaArgs.length) {
-      spawnArgs = spawnArgs.concat(convertedMochaArgs);
+      spawnArgs = spawnArgs.concat(convertedMochaArgs).concat(nodeArgs);
```

Comment:
This is nitpicky, but you could have just passed a second argument to the first call to `concat` instead of calling `concat` twice.

Reply:
Ah yeah. I'll clean that up.